CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/crt/aws-c-common/cmake"
 include(AwsFindPackage)
 
 set(IN_SOURCE_BUILD ON)
+set(SEARCH_LIBCRYPTO OFF CACHE BOOL "Let S2N use libcrypto from AWS-LC.")
```

Comment:
Shouldn't this be ON? FFI has been using aws-lc since it came out.

Reply:
don't know, got this from Pushen, after all the CI's were failing on missing libcrypto. Now CI passes.

Reply:
The comment/description feels inverted. We'll YOLO it for now, but we need to clarify what's up with that.

Reply:
@wps132230

Reply:
The flag is a little bit misleading...
`SEARCH_LIBCRYPTO=ON` means it's actually doing the "search", and try to find the pre-installed libcrypto with `FindLibCrypto.cmake`.
When this flag is OFF, it will use the `crypto` target defined in AWS-LC or any alternative instead of searching for it in any location.
https://github.com/aws/s2n-tls/blob/7f43b102def1d52422f6c3e48d5cb3e6dd26c646/CMakeLists.txt#L345-L353